app/uk/gov/hmrc/apinotificationqueue/model/notificationModels.scala

```diff
@@ -37,6 +37,11 @@ object NotificationWithIdOnly {
   implicit val notificationWithIdOnlyJF = Json.format[NotificationWithIdOnly]
 }
 
+case class NotificationWithIdAndPulledStatus(notification: NotificationId, pulledStatus: Boolean)
```

Comment:
Call the parameter just `pulled` that way it reads as `has been pulled` when true

Reply:
Done